### PR TITLE
Add tomato vs cucumber vs beer classifier model

### PR DESCRIPTION
**Pull request by EdvardasDlugauskas**
It can tell whether the image is a tomato or a cucumber

### CODE REVIEW

**Comment by EdvardasDlugauskas** [comment]
Could somebody try to run the project and tell me if it works?

If you run into problems on a mac, follow the README.md file in the project root.

File: ShopLens/ImageRecognition/Program.cs [hunk 1]
```diff
@@ -10,13 +10,13 @@ namespace ImageRecognition
...
-            "C:\\Users\\Edvard\\RiderProjects\\TOP2018\\ShopLens\\ImageRecognition\\resources\\train\\tomato\\1.jpg";
+            "resources\\test\\2.jpg";
 
         public static string TensorFlowLabelsFilePath { get; private set; } =
-            "C:\\Users\\Edvard\\RiderProjects\\TOP2018\\ShopLens\\ImageRecognition\\resources\\model\\labels.txt";
+            "resources\\model\\labels.txt";
 
         public static string TensorFlowModelFilePath { get; private set; } =
-            "C:\\Users\\Edvard\\RiderProjects\\TOP2018\\ShopLens\\ImageRecognition\\resources\\model\\model.pb";
+            "resources\\model\\model.pb";
```
Comment (marjasi) [inline ShopLens/ImageRecognition/Program.cs]: For some reason the program 'can't find a part of the path' to all the required files on my machine. It should work and everything seems fine. Weird.

File: ShopLens/ImageRecognition/resources/model/labels.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
+beer
```
Comment (marjasi) [inline ShopLens/ImageRecognition/resources/model/labels.txt:1]: Nice addition.

File: ShopLens/ImageRecognition/Program.cs [hunk 3]
```diff
@@ -1,12 +1,87 @@
...
     class Program
     {
+        public static string TestImageFilePath { get; private set; } =
+            "resources\\test\\2.jpg";
+
+        public static string TensorFlowLabelsFilePath { get; private set; } =
+            "resources\\model\\labels.txt";
+
+        public static string TensorFlowModelFilePath { get; private set; } =
+            "resources\\model\\model.pb";
```
Comment (marjasi) [inline ShopLens/ImageRecognition/Program.cs:19]: I even tried changing it to "..\\resources\\model\\model.pb" and it still couldn't find the path.

File: ShopLens/ImageRecognition/TfImageUtil.cs [hunk 4]
```diff
@@ -0,0 +1,81 @@
+﻿// Author: Miguel de Icaza
+// Taken and adapted from: https://github.com/migueldeicaza/TensorFlowSharp/blob/master/Examples/ExampleCommon/ImageUtil.cs
```
Comment (marjasi) [inline ShopLens/ImageRecognition/TfImageUtil.cs:2]: Maybe we should add this library and credit the author on our master branch README.md?

**Review by marjasi — CHANGES_REQUESTED** [review]
Great work and humorous beer addition! However, the project doesn't run on my machine because it can't find the paths to the resource files.

File: ShopLens/ImageRecognition/Program.cs (hunk 3 above)
Comment (MellowJay) [reply ShopLens/ImageRecognition/Program.cs]: I mean "..\\resources\\model\\model.pb" to be exact.

**Review by marjasi — APPROVED** [review]

**Pull request merged**